lib/rucio/web/ui/static/did.js

```diff
@@ -99,6 +100,7 @@ load_replicas = function(scope, name) {
     r.list_replicas({
         'scope': scope,
         'name': name,
+	'schemes': ['davs'],
```

Comment:
I'm afraid you will have to do this call twice. If you do it like this only replicas which are on an RSE that supports davs are listed. But in the table it should always show all available replicas and then mark those that are on an RSE that is browser enabled.